Doc/library/functions.rst

```diff
@@ -61,6 +61,21 @@ are always available.  They are listed here in alphabetical order.
    If the argument is a complex number, its magnitude is returned.
 
 
+.. function:: aiter(async_iterable)
+
+   Return an :term:`asynchronous iterator`. This is the async variant
+   of the :func:`iter` builtin, and behaves similarly.
+
+   *async_iterable* must be an :term:`asynchronous iterable`,
+   and :func:`aiter` returns an asynchronous iterator for it.
+
+   Unlike the :func:`iter` builtin, :func:`aiter` has no 2-argument variant.
+   Often, this variant can be replaced with assignment expressions::
+
+      while chunk := await sock.read(CHUNK_SIZE):
+         ...
```

Comment:
Removed.

Reply:
One reason I quite like this snippet is that it helps to avoid surprise at why `aiter()` doesn't have a 2-argument form to match `iter()`. But it does feel out of place to talk about things this function _doesn't do_.